osgar/lib/serialize.py

```diff
@@ -0,0 +1,20 @@
+"""
+  serialization of messages
+"""
+from ast import literal_eval
+
+
+def serialize(data):
+    try:
+        bytes_data = data.tobytes()
+    except AttributeError:
+        if isinstance(data, bytes):
+            bytes_data = data
+        else:
+            bytes_data = bytes(str(data), encoding='ascii')
+    return bytes_data
+
+
+def deserialize(bytes_data):
+    return literal_eval(bytes_data.decode('ascii'))
```

Comment:
So in order for `serialize` to be the opposite of this `deserialize`, it would have to be something like this:

```python
def serialize(data):
  return bytes(str(data), encoding='ascii'))
```
Do I understand the code correctly? That would be "the eval solution", right?